Sources/Workspace/UserToolchain.swift

```diff
@@ -22,6 +22,18 @@ import Utility
     private let whichClangArgs = ["which", "clang"]
 #endif
 
+#if os(macOS)
```

Comment:
I think we should find a way to avoid the duplication here, also for the `getLlvmCov()` method etc. We can make the cache of paths a dictionary and have one generic method that gets the binary name as an argument. The public interface can stay the same but then we don't need three copies of the same method essentially.

Reply:
done